tests/cpp/dlr_data_transform_test.cc

```diff
@@ -225,7 +225,7 @@ TEST(DLR, DataTransformDateTime) {
   }
 }
 
-TEST(DLR, RelayVMDataTransformInput) {
+TEST(DLR, DISABLED_RelayVMDataTransformInput) {
```

Comment:
Lets add a reason, recommendations and TODO label in a new comment above

Reply:
Done